### ISSUE TITLE
C++/CLI support on .NET 5 for ARM64

### ISSUE THREAD

**Issue by Alovchin91** [dotnet/core#4975]
Are there any plans to support C++/CLI on .NET 5 for ARM64 platforms? If yes, is there any timeline for this support?

One of the important use cases for it would be Paint.NET: https://forums.getpaint.net/topic/116030-feature-request-arm64-builds/

**Comment by scalablecory** [comment]
@richlander @terrajobst

**Comment by Alovchin91** [comment]
@scalablecory @richlander @terrajobst  Any chance to get an idea a month later? 😅

**Comment by davidfowl** [comment]
AFAIK we have no plans to do this:

cc @jeffschwMSFT

**Comment by Alovchin91** [comment]
@davidfowl  Thanks for the response. Is it no plans at all, or no plans for .NET 5 but maybe in the future?

**Comment by terrajobst** [comment]
C++/CLI is legacy tech, we don't plan on innovating in it. I wouldn't rule out supporting it on ARM, but it's a function of business needs. Unless there is a strong need, the answer is likely no.

**Comment by tgani-msft** [comment]
We have implemented support for C++/CLI on ARM64 .NET Core. However, currently, it is "compiler+libs only", with debugger and project system support still under development. Our guidance for using C++/CLI on .NET Core will remain the same as before: use C++/CLI in a sub-project of a C# .NET Core project.

@terrajobst is correct in the legacy nature of C++/CLI and we have not done any further development of the language to catch up with newer .NET features. It remains, and will remain for the foreseeable future, as an interop language between native and .NET code. This was the primary reason we implemented support for C++/CLI on .NET Core.
